### ISSUE TITLE
Remove accessibleText

### ISSUE THREAD

**Issue by marksreeves** [BorderTech/wcomponents#374]
The accessibleText property and accessors on `AbstractWComponent` were introduced to work around a perceived accessibility issue due to a misunderstanding of WCAG 2.0. Recent clarification of these guidelines make accessibleText more of a problem than a solution therefore it should be removed.

**Comment by marksreeves** [comment]
On second thoughts we can reuse it to make aria-label where required. Still a poor cousin in the labelling family tree but not totally worthless. Closed invalid.
